checkout/frames-ios
# Implement security code component UI

## Issue

JIRA-2167

## Proposed changes

Add Security Code Component UI.

## Test Steps

> 1. Go to the main screen
> 2. Tap on CVV Component button
> 3. Verify the screen is visible and the validation works

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [x] I have added UI tests that prove my fix is effective or that my feature works
* [N/A] I have added regression tests that prove my fix is effective or that my feature works

Just having a set of basic UITests felt enough. No need for detailed regression tests. If you see some scenarios that'd do well with extensive regression tests being written upon, please add a comment to this PR.
